SproxydKeysScan/runSproxydKeysScan.js

```diff
@@ -0,0 +1,85 @@
+const { RaftJournalReader } = require('./DuplicateKeysIngestion');
+const { getSproxydAlias } = require('../repairDuplicateVersionsSuite');
+const { Logger } = require('werelogs');
+
+const log = new Logger('s3utils:SproxydKeysScan:run');
+
+const env = {
+    SPROXYD_KEY_LIMIT: process.env.SPROXYD_KEY_LIMIT,
+    RAFT_SESSION_ID: process.env.RAFT_SESSION_ID,
+    LOOKBACK_WINDOW: process.env.LOOKBACK_WINDOW,
+};
+
+const USAGE = `
+runSproxydKeyScan.js
```

Comment:
```suggestion
runSproxydKeysScan.js
```

Also, the "run" prefix looks redundant or too vague since all scripts can be "run", or maybe, call it `sproxydKeysScanDaemon` to make it explicit it is a daemon.